packages/where/index.js

```diff
@@ -59,32 +95,34 @@ export const whereEndoSock = (platform, env) => {
     } else {
       return raw`\\?\pipe\Endo\endo.pipe`;
     }
+  } else if (env.XDG_RUNTIME_DIR !== undefined) {
+    return `${env.XDG_RUNTIME_DIR}/endo/endo.sock`;
   } else if (platform === 'darwin') {
     if (env.HOME !== undefined) {
       return `${env.HOME}/Library/Application Support/Endo/endo.sock`;
     }
-  } else if (env.XDG_RUNTIME_DIR !== undefined) {
-    return `${env.XDG_RUNTIME_DIR}/endo/endo.sock`;
   } else if (env.USER !== undefined) {
     return `/tmp/endo-${env.USER}/endo.sock`;
   }
-  return 'endo.sock';
+  return 'endo/endo.sock';
```

Comment:
- above comments imply this branch goes away, since:
  - $USER is always defined or definable via `getent(1)`
  - but do consult `$TMPDIR` here

Reply:
I’m using TMPDIR and falling back to `os.tmpdir()` now. Thanks.